payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/Payroll.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace PayrollProcessor.Functions.Features.Payrolls
+{
+    public class Payroll
+    {
+        public Guid Id { get; set; }
```

Comment:
I should probably make this nullable or create a new `PayrollNew` type that doesn't have the property to make it clearer that the `Id` is generated by the Function, not the client.

Reply:
We can come back to this when the client starts creating payrolls